datasets/pass/metadata.json

```diff
@@ -3,10 +3,17 @@
     "@vocab": "https://schema.org/",
     "sc": "https://schema.org/",
     "ml": "http://mlcommons.org/schema/",
-    "includes": {
-      "@id": "ml:includes",
-      "@type": "@id"
-    }
+    "includes": "ml:includes",
```

Comment:
Do we need to define those somewhere?

Reply:
For the moment, all constants are defined in python/format/src/constants.py.

`"includes": "ml:includes"` is a context shortcut to map `"includes"` to `"http://mlcommons.org/schema/includes"`.

Did you mean anything different?